maptivist/src/main/webapp/script.js

```diff
@@ -55,8 +55,15 @@ function initMap(){
         <a href="https://en.wikipedia.org/wiki/George_Floyd">Related source</a>
         <br>
         <div class="upvote">
-            <button>Upvote</button>
             <p>counter: </p>
+            <form action="/marker" method="POST">
```

Comment:
I suggest creating a new servelet to handle upvotes ("/upvote"). Since that's a lot more work, perhaps its better to do that is a separate PR?